docs/content/docs/try-flink-ml/java/quick-start.md

```diff
@@ -28,18 +28,25 @@ under the License.
 
 This document provides a quick introduction to using Flink ML. Readers of this
 document will be guided to submit a simple Flink job that trains a Machine
-Learning Model and use it to provide prediction service.
+Learning Model and uses it to provide prediction service.
 
-## Prerequisites
+## Help, I’m Stuck!
 
-### Install Flink
+If you get stuck, check out the [community support
+resources](https://flink.apache.org/gettinghelp.html). In particular, Apache
+Flink's [user mailing
+list](https://flink.apache.org/community.html#mailing-lists) is consistently
+ranked as one of the most active of any Apache project and a great way to get
+help quickly.
+
+## Install Flink
```

Comment:
The link given below does not mention anything related to `Install Flink`. 

Do you actually mean `Download Flink` here? If so, it seems simpler to just copy/paste the related text instead of redirecting readers to another page.